v2.6/usage/decommissioning-a-node.md

```diff
@@ -11,9 +11,15 @@ redirect_from: latest/usage/decommissioning-a-node
 - You are receiving an error about an IP address already in use.
 - Hosts are regularly added and removed from your cluster.
 
+### Why you might not be interested in this guide
+
+- You are running Calico in Kubernetes-Datastore Mode, which automatically
+handles node deletion.
+- You are running the Calico Policy Controller v1.0.1+, which automatically handles node deletion.
+
 ### Purpose of this page
 
-Provide guidance on how to remove a host that is part of a Calico cluster
+Provide guidance on how to manually remove a host that is part of a Calico cluster
```

Comment:
I'd undo this change - the information in this doc can still apply to homespun automated processes.